ts/screens/wallet/payment/ScanQrCodeScreen.tsx

```diff
@@ -171,7 +206,13 @@ class ScanQrCodeScreen extends React.Component<Props, State> {
           .catch(() => {
             this.onInvalidQrCode();
           });
-      }
+      } else if (response.error !== undefined && Platform.OS === "ios") {
```

Comment:
Why only on iOS?
Using Android if I reject (**and don't ask again**) to allow reading media, I get the same issue.
![android-bug](https://user-images.githubusercontent.com/822471/73374166-085ec400-42ba-11ea-852f-1cb5b8442efd.gif)

Reply:
@Undermaken Because for android the message is taken from here https://github.com/teamdigitale/io-app/blob/6e0ddbdc2f4643111ad87bdad856aab7108a892e/ts/screens/wallet/payment/ScanQrCodeScreen.tsx#L168

Reply:
ok. Anyway when Android asks for permission and you choose **"No and don't ask it again (Rifiuta e non chiedermelo più)"** and then you tap the button "pick image from gallery" nothing happens.

Instead if you remove the condition `Platform.OS === "ios"` an info alert will be shown and you can open the settings to torn un the permission.
I guess this last behavior is right

Reply:
@Undermaken I had entered the control on ios to avoid the double consecutive view of the alert (Android only), now I have removed it and this is the result

![double_alert](https://user-images.githubusercontent.com/7993551/73740039-ada8ea80-4747-11ea-8f47-687d53ba9260.gif)